### PR TITLE
3.x

### PR DESCRIPTION
**Pull request by alesmenzel**
This is a breaking change that supports passport version `0.6.x` - see changelog https://github.com/jaredhanson/passport/blob/1e8f112bd233dbffb1904d4dd2051780d81b0a22/CHANGELOG.md#060---2022-05-20

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -60,8 +60,10 @@
   },
   "devDependencies": {
     "@types/debug": "^4.1.7",
+    "@types/express-session": "^1.17.4",
```
Comment (cjbarth) [inline package.json:63]: I believe all of these should actually be `dependencies` not `devDependencies`.

**Review by cjbarth — CHANGES_REQUESTED** [review]
Thank you so much for your contribution! I noted this and was trying to make time to address it.

As I started reviewing this, I noticed that this isn't against `master`. Since this will be a breaking change, it should be against `master` where all the other 4.x changes are. Please make this against `master` before I continue reviewing it.

File: package.json (hunk 1 above)
Comment (alesmenzel) [reply package.json:63]: Hi, the express-session is only used in tests. I will close this PR and reopen it on master.

**Comment by alesmenzel** [comment]
Updating master instead -> https://github.com/node-saml/passport-saml/pull/698

**Pull request closed without merging**